src/resources/platform-adaptations.md

```diff
@@ -739,7 +739,73 @@ Scaffold(
                   .toList(),
             ));
 ```
+### Text fields
 
+Since Android 12, the default UI of text fields follows the design 
+guidelines defined in [Material 3][m3-text-field] (M3). 
+On iOS, an equivalent component is defined in 
+[Human Interface Guidelines][hig-text-field] (HIG). 
```

Comment:
**nit:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
On iOS, Apple's [Human Interface Guidelines][hig-text-field] (HIG) define
an equivalent component. 
```